### PR TITLE
Dependencies no linting

### PR DESCRIPTION
**Pull request by ylvaselling**
Updated dependencies

### CODE REVIEW

File: src/components/common/MaterialIcon/MaterialIcon.scss [hunk 1]
```diff
@@ -38,11 +43,10 @@
...
 
-
 /* Rules for using icons as black on a light background. */
 .base.md-dark { color: rgba(0, 0, 0, 0.54); }
 .base.md-dark.md-inactive { color: rgba(0, 0, 0, 0.26); }
 
 /* Rules for using icons as white on a dark background. */
 .base.md-light { color: rgba(255, 255, 255, 1); }
-.base.md-light.md-inactive { color: rgba(255, 255, 255, 0.3); }
+.base.md-light.md-inactive { color: rgba(255, 255, 255, 0.3); }
```
Comment (WeirdRubberDuck) [inline src/components/common/MaterialIcon/MaterialIcon.scss:52]: MIssing new line

File: src/index.jsx [hunk 2]
```diff
@@ -19,20 +18,10 @@ const store = createStore(
...
-if (module.hot) {
-  module.hot.accept('./App', () => render(App));
-}
+const container = document.getElementById('root');
+const root = createRoot(container);
+root.render(
+  <Provider store={store} >
+    <App />
+  </Provider>
+);
```
Comment (WeirdRubberDuck) [inline src/index.jsx:27]: MIssing new line

File: src/views/OnScreenGui.jsx [hunk 3]
```diff
@@ -19,123 +21,110 @@ import styles from './OnScreenGui.scss';
...
-        <TutorialProvider>
-          {this.props.showAbout && (
+  return (
+    <div className={styles.app}>
+      <TutorialProvider>
+        { showAbout && (
           <Overlay>
             <Stack style={{ maxWidth: '500px' }}>
-              <Button style={{ alignSelf: 'flex-end', color: 'white' }} onClick={this.props.hideAbout}>
+              <Button style={{ alignSelf: 'flex-end', color: 'white' }} onClick={hideAbout}>
```
Comment (WeirdRubberDuck) [inline src/views/OnScreenGui.jsx:77]: line width 

I'd argue that even in the GUI code we should try to aim for a maximum line width of 90 characters

I know that there are already a lot of places in the code where the lines are already too long... But why not leave the code better than we found it when doing changes :)

File: src/views/OnTouchGui.jsx [hunk 4]
```diff
@@ -31,52 +27,60 @@ import {
...
-
-    this.addStoryTree = this.addStoryTree.bind(this);
-    this.changeStory = this.changeStory.bind(this);
-    this.setStory = this.setStory.bind(this);
-    this.resetStory = this.resetStory.bind(this);
-    this.checkStorySettings = this.checkStorySettings.bind(this);
-    this.handleKeyPress = this.handleKeyPress.bind(this);
-    this.toggleDeveloperMode = this.toggleDeveloperMode.bind(this);
-  }
+function OnTouchGui({ luaApi, FetchData, StartConnection, anchorNode, changePropertyValue, triggerActionDispatcher,
```
Comment (WeirdRubberDuck) [inline src/views/OnTouchGui.jsx:30]: line width 

(see comment above)

Comment (WeirdRubberDuck) [reply src/views/OnTouchGui.jsx:30]: next line too

**Review by WeirdRubberDuck — CHANGES_REQUESTED** [review]
Code looks good to me and I had noi problem running the regular onscreen GUI, but there are errors when opening the remote and touch GUI. These should be fixed. 

For example, when opening the touch GUI in the browser:
image

The same goes for the remote GUI, which has the endpoint: "http://127.0.0.1:4690/frontend/#/remote"



Note that these problems probably emerged from the Getting started guide tutorial merge and not the dependency update. I get the same errors in master. So if you prefer, you could fix them in the master branch and leave this PR as it is (assuming this PR still runs well after the fix).

File: src/views/OnTouchGui.jsx (hunk 4 above)
Comment (ylvaselling) [reply src/views/OnTouchGui.jsx:30]: We could also apply line width as a rule in the linter?

File: src/components/common/MaterialIcon/MaterialIcon.scss (hunk 1 above)
Comment (WeirdRubberDuck) [reply src/components/common/MaterialIcon/MaterialIcon.scss:52]: Probably also something that can be included in the linter
https://eslint.org/docs/latest/rules/eol-last

**Review by WeirdRubberDuck — APPROVED** [review]
Everything seems to run fine now :)

**Pull request merged**